src/nyc_trees/apps/users/models.py

```diff
@@ -89,13 +89,10 @@ def __init__(self, name, description, description_achieved, badge,
         badge='img/badges/ic_badge_map_50.png',
         achieved=lambda user: user.blocks_mapped_count >= 50,
         sponsor='Various sponsors',
-        reward='The first 250 volun<b>treers</b> to reach '
-                'this level will receive tickets to a NY '
-                'Liberty Game! All volun<b>treers</b> will '
-                'be entered into lotteries to attend a '
-                'Picnic in the Park, win a personal '
-                'Tree Care Workshop, a Knicks gift '
-                'bag and other great prizes while supplies last.'
+        reward='The first 250 voluntreers to reach this level will '
+        'receive tickets to a NY Liberty Game! All voluntreers will '
```

Comment:
I assume we're purposefully removing the `<b>` around "treer"?

If so, can you remove the `|safe` template filter here: https://github.com/azavea/nyc-trees/blob/develop/src/nyc_trees/apps/users/templates/users/partials/achievement_modal.html#L6?